library/src/scripts/headers/pieces/MobileDropDown.tsx

```diff
@@ -73,51 +73,53 @@ export default class MobileDropDown extends React.Component<IProps, IState> {
+                                                className={classNames(
+                                                    "frameHeader-leftSpacer",
+                                                    classesFrameHeader.leftSpacer,
+                                                )}
                                             />
+                                            <Heading
+                                                title={title}
+                                                className={classNames(
+                                                    "frameHeader-heading",
+                                                    "frameHeader-centred",
+                                                    classesFrameHeader.centred,
+                                                    classesFrameHeader.heading,
+                                                )}
+                                            >
+                                                <SmartAlign>{title}</SmartAlign>
+                                            </Heading>
+                                            <div
+                                                className={classNames(
+                                                    "frameHeader-closePosition",
+                                                    classesFrameHeader.action,
+                                                )}
+                                            >
+                                                <CloseButton
+                                                    className={classNames(classes.closeButton)}
+                                                    onClick={this.close}
+                                                    compact={true}
+                                                />
+                                            </div>
                                         </div>
-                                    </header>
-                                    <FrameBody className={this.props.frameBodyClassName}>{children}</FrameBody>
-                                    <FrameFooter>
-                                        <Button
-                                            onClick={this.close}
-                                            baseClass={ButtonTypes.CUSTOM}
-                                            className={classes.closeModal}
-                                        >
-                                            {chevronUp(classes.closeModalIcon)}
-                                        </Button>
-                                    </FrameFooter>
-                                </Frame>
-                            </Panel>
+                                    </PanelWidgetHorizontalPadding>
+                                </Container>
+                            </header>
+                            <FrameBody className={this.props.frameBodyClassName}>{children}</FrameBody>
```

Comment:
Since this has overflow: "auto" on it, it needs a wrapper of `<TouchScrollable />`. I'd recommend giving this a test in browser stack.

Reply:
![image](https://user-images.githubusercontent.com/1770056/56225058-ad1b5800-603e-11e9-8f20-ef61bbbe91db.png)

Example from my real iOS device. Nothing here is scrollable.